internal/pkg/scaffold/cmd.go

```diff
@@ -159,4 +161,14 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func serveCRMetrics(cfg *rest.Config) {
+	operatorSpecificScheme := run.NewScheme()
+	apis.AddToScheme(operatorSpecificScheme)
+	filteredScheme := kubemetrics.FilterOutMetaTypes(operatorSpecificScheme)
+	err := kubemetrics.ServeCRMetrics(cfg, []string{}, filteredScheme, metricsHost, metricsOperatorPort)
```

Comment:
@joelanford Currently this generates very specific metrics for the custom resources. But users could want to use this to generate and serve their own custom resource metrics, for example, the number of replicas. 

How do we want to approach this, I can see the following way, but it means a bit more code on the users side:
```go
collectors, err := kubemetrics.GenerateCRMetrics(cfg, []string{}, filteredScheme)
...
// user could create their own collectors based on their generated family metrics and GVK they are interested in.
// they append the collector to the above-generated collectors
go kubemetrics.ServeMetrics(collectors, metricsHost, metricsOperatorPort)
```

This would just mean we take out the serving out the serving out of `kubemetrics.ServeCRMetrics() ` function and move it into the `main.go`, but would allow users to add more metrics in the future. Otherwise the user would need to serve metrics on yet another port.